target/linux/lantiq/image/Makefile

```diff
@@ -487,9 +515,10 @@ TARGET_DEVICES += buffalo_wbmr-hp-g300h-b
 
 define Device/avm_fritz7312
   $(Device/AVM)
```

Comment:
You can use the parent definition "AVM" to hold the DEVICE_VENDOR for all AVM devices.